test/shopify-cli/project_type_test.rb

```diff
@@ -32,5 +32,11 @@ def test_project_filepath
         File.join(ShopifyCli::PROJECT_TYPES_DIR, 'rails', 'myfile')
       )
     end
+
+    def test_duplicate_command
+      assert_raises ShopifyCli::Abort do
```

Comment:
Would it be worth checking the message here to ensure it's aborting for the right reason?

Reply:
Fair point!